### ISSUE TITLE
remove deprecated distutils

### ISSUE THREAD

**Issue by zerwes** [zerwes/hiyapyco#61]
```
hiyapyco/__init__.py:25:0: W4901: Deprecated module 'distutils.util' (deprecated-module)
```
caused by `strtobool`

https://peps.python.org/pep-0632/#migration-advice

so we can use a own implementation or use:
 * https://pypi.org/project/str2bool/
 * https://github.com/symonsoft/str2bool

**Comment by zerwes** [comment]
minimal own implementation would look like
```python
def strtobool(val):
    val = val.lower()
    if val in ("y", "yes", "t", "true", "on", "1"):
        return 1
    elif val in ("n", "no", "f", "false", "off", "0"):
        return 0
    else:
        raise ValueError("invalid truth value %r" % (val,))
```
! strtobool returns int, not bool :-?
```python
>>> distutils.util.strtobool('y')
1
>>> type(distutils.util.strtobool('y'))
<class 'int'>
```

**Comment by zerwes** [comment]
fixed in PR #70

**Comment by zerwes** [comment]
release 0.5.6 including the fixes from pr #70 has been released
